indico/modules/events/util.py

```diff
@@ -501,3 +502,10 @@ def serialize_event_for_ical(event, detail_level):
     if detail_level == 'contributions':
         data['contributions'] = [serialize_contribution_for_ical(c) for c in event.contributions]
     return data
+
+
+def render_acl(obj):
+    acl = obj.get_access_list(skip_managers=True, skip_self_acl=True)
+    users = [x for x in acl if x.principal_type in (PrincipalType.user, PrincipalType.email)]
+    groups = [x for x in acl if x.principal_type in {PrincipalType.multipass_group, PrincipalType.local_group}]
+    return jsonify_template('forms/_access_list.html', users=users, groups=groups)
```

Comment:
I think it'd be a better fit directly in `indico/web/templates/` instead of `indico/web/templates/forms/`